javascript-practice/src/styles/base/typography.css

```diff
@@ -0,0 +1,13 @@
+@font-face {
+  font-family: 'inter';
+  src: url('../../assets/fonts/Inter/Inter-Regular.woff2') format('woff2');
+  font-weight: 400;
+  font-display: swap;
+}
+
+@font-face {
+  font-family: 'inter';
+  src: url('../../assets/fonts//Inter/Inter-Medium.woff2') format('woff2');
```

Comment:
Wrong the path `fonts//Inter/`. Should use lowercase for the folder name. Should be `inter`